### PR TITLE
Adds keyboard input logic to game component

### PR DESCRIPTION
**Pull request by nlaveck**

### CODE REVIEW

File: src/components/game.tsx [hunk 1]
```diff
@@ -15,12 +35,15 @@ const Game: FC = () => {
...
 
-
+    console.log(input);
+    
     return (
     <div className="flex-center column">
         { renderRows(ROW_COUNT) }
     </div>
     );
+
```
Comment (lunchkin) [inline src/components/game.tsx:45]: TWO BLANK LINES?!?!?!?!?!

File: src/components/game.tsx [hunk 2]
```diff
@@ -15,12 +35,15 @@ const Game: FC = () => {
         return rows;
     };
 
-
+    console.log(input);
+    
     return (
     <div className="flex-center column">
```
Comment (lunchkin) [inline src/components/game.tsx:41]: Indent much?! Got tabs?!

**Review by acantrell — APPROVED** [review]
:shipit:

File: src/components/game.tsx [hunk 3]
```diff
@@ -1,11 +1,30 @@
...
+
     const ROW_COUNT = 6;
+    const WORD_LENGTH = 5;
+
+    useEffect(() => {
+        const onKeyDown = ({key }) => {
+            // console.log(key);
+            if (key === "Backspace") {
+                setInput(input.slice(0, input.length - 1))
+            } else if (input.length < WORD_LENGTH) {
```
Comment (lunchkin) [inline src/components/game.tsx:18]: got em!

**Pull request merged**